config/02-api/22-api-deployment.yaml

```diff
@@ -41,6 +41,13 @@ spec:
       containers:
         - name: tekton-hub-api
           image: quay.io/tekton-hub/api
+          resources:
+            requests:
+              cpu: 100m
+              memory: 100Mi
+            limits:
+              cpu: 500m
+              memory: 500Mi
```

Comment:
are these sufficient?

Reply:
on what basis are we providing limits?
is it even required?